main.js

```diff
@@ -186,28 +217,149 @@ function toggleSelectElement(event) {
     })
 }
 
-/* Helpers */
+/*
+****************************************************** Layout ******************************************************
+*/
 
-function isTitleVisible() {
-    return titleVisibleToggle.textContent === "Set Hidden"
+function createLayout(items, gridWidth, gridHeight) {
+    // The layout data object. Muuri will read this data and position the items
+    // based on it.
+    var layout = {
+      // The layout item slots (left/top coordinates).
+      slots: [],
+      // The layout's total width.
+      width: itemWidth,
+      // The layout's total height.
+      height: itemHeight,
+      // Should Muuri set the grid's width after layout?
+      setWidth: true,
+      // Should Muuri set the grid's height after layout?
+      setHeight: true
+    };
+
+    // Calculate the slots.
+    
+    const numberOfColumns = columnInput.value;
+    const numberOfRows = rowInput.value;
+    const margin = 5;
+
+    let item;
+    let x = margin;
+    let y = 0;
+
+    let width = itemWidth;
+    let height = itemHeight;
```

Comment:
Are width and height being modified anywhere? If they aren't then they could be made into `const` instead of `let`

Reply:
True, I made them let due to Swift habits, changed them to const now.